src/metabase/api/user.clj

```diff
@@ -297,7 +297,8 @@
   ;; only allow updates if the specified account is active
   (api/let-404 [user-before-update (fetch-user :id id, :is_active true)]
     ;; Google/LDAP non-admin users can't change their email to prevent account hijacking
-    (api/check-403 (valid-email-update? user-before-update email))
+    (api/check-403 (or (not (contains? body :email))
```

Comment:
If we're not updating email, skip the `valid-email-update?` check, otherwise, it would throw **You don't have permissions to do that.**

Reply:
Trying this locally and I don't think we need this change at all.
If I remove the change and change the locale via the UI, the change is still accepted.

Perhaps I'm not testing the right thing? But, if you try locally, consider taking this change out to keep things simpler.

Reply:
I swear I hit the `You don't have permissions to do that.` thing without this change yesterday. However, I just retested this, and seems like it can be inverted. Thanks for the feedback.